MapboxNavigation/LanesView.swift

```diff
@@ -70,14 +70,22 @@ open class LanesView: UIView {
         clearLaneViews()
         
         let step = currentLegProgress.currentStep
-        guard let firstInstruction = step.instructionsDisplayedAlongStep?.first,
-              let lanes: [LaneIndicationComponent] = firstInstruction.tertiaryInstruction?.components.compactMap({ component in
-                guard let lane = component as? LaneIndicationComponent else { return nil }
-                return lane
-              }), !lanes.isEmpty
-            else {
-                hide()
-                return
+        guard let instructionsDisplayedAlongStep = step.instructionsDisplayedAlongStep else {
+            hide()
+            return
+        }
+        
+        let components: [ComponentRepresentable] = instructionsDisplayedAlongStep
```

Comment:
@vincethecoder I'm still not seeing how we're finding the current visual instruction that should be displayed. If a step has 3 different tertiary instructions, how do we know which one to show?

Reply:
@bsudekum smh - using the current spoken instruction index. Will update in a moment.